android/build.gradle

```diff
@@ -19,14 +18,12 @@ ext {
 
 allprojects {
     repositories {
-        jcenter()
         google()
     }
 }
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion '28.0.3'
+    compileSdkVersion 30
```

Comment:
Shouldn't we also update the `targetSdkVersion` to match the `compileSdkVersion`? :thinking:

Reply:
No. Not at all. Very different concepts.